### ISSUE TITLE
connect to remote K8S

### ISSUE THREAD

**Issue by arcturuscom** [InfoSec812/vertx-configmap-example#4]
please can you provide an exemple of how to connect to a remote K8S, i'm using EKS

**Comment by InfoSec812** [comment]
I don't believe you CAN connect to a remote K8s server. It has to run inside of the cluster.
